content/en/docs/tasks/administer-cluster/dns-custom-nameservers.md

```diff
@@ -8,13 +8,16 @@ content_template: templates/task
 
 {{% capture overview %}}
 This page explains how to configure your DNS Pod and customize the
-DNS resolution process.
+DNS resolution process. In Kubernetes version 1.11 and later, CoreDNS is installed
+by default. See [Configuring CoreDNS](#config-coredns) and 
```

Comment:
instead of:
`In Kubernetes version 1.11 and later, CoreDNS is installed by default.`

we need to say something in the lines of:
`In Kubernetes version 1.11, CoreDNS has reached the General Availability status and is installed by default from kubeadm.`

so it is GA since 1.11, but only installed by default from kubeadm.